@planetarium/tx/src/binary.ts

```diff
@@ -0,0 +1,8 @@
+
+export function compareUint8Array(a: Uint8Array, b: Uint8Array) {
+  const length = Math.min(a.length, b.length);
+  for (let i = 0; i < length; i++) {
+    if (a[i] !== b[i]) return a[i] - b[i];
```

Comment:
Just a curiosity, but is the reason for returning the difference of two bytes about interpreter optimization?

Reply:
Actually there's no reason behind this.  It's just my habit.